content/en/docs/reference/labels-annotations-taints/audit-annotations.md

```diff
@@ -56,4 +56,20 @@ that was transgressed as well as the specific policies on the fields that were
 violated from the PodSecurity enforcement.
 
 See [Pod Security Standards](/docs/concepts/security/pod-security-standards/)
-for more information.
\ No newline at end of file
+for more information.
+
+## authorization.k8s.io/decision
+
+Example: `authorization.k8s.io/decision: {allow, forbid}`
+
+This annotation indicates whether or not a request was authorized in Kubernetes audit logs.
+
+See [Auditing](/blog/2021/12/09/pod-security-admission-beta/#auditing) for more information.
```

Comment:
I would link to https://kubernetes.io/docs/tasks/debug-application-cluster/audit/ rather than to that blog article.

Reply:
done.